dll/shellext/shellbtrfs/reactos.cpp

```diff
@@ -153,12 +153,24 @@ NTSTATUS WINAPI RtlUTF8ToUnicodeN(PWSTR uni_dest, ULONG uni_bytes_max,
 }
 
 /* Quick and dirty table for conversion */
-FILE_INFORMATION_CLASS ConvertToFileInfo[MaximumFileInfoByHandlesClass] =
+#if 0 // < VISTA
+#ifndef FileIoPriorityHintInformation // FILE_INFORMATION_CLASS NT6.0+ feature.
+#define FileIoPriorityHintInformation ((FILE_INFORMATION_CLASS)43)
+#endif
+
+// FILE_INFO_BY_HANDLE_CLASS and its MaximumFileInfoByHandleClass are NT6.0+.
+// Some other files depend on NT6.0+ too, so to be continued later...
```

Comment:
// Some other files depend on NT6.0+ too, so to be continued later...
That comment belongs in the PRs description, but not into the code.

Reply:
No: description reads 'Remove ShellBtrfs commit, which is there to give context only.'